### PR TITLE
feature(kevin): add Turkish support

### PR DESCRIPTION
**Pull request by ruoyuGao**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -312,10 +312,14 @@ format:
 - [YouTube] [John Schulman - Reinforcement Learning from Human Feedback: Progress and Challenges](https://www.youtube.com/watch?v=hhiLw5Q_UFg)
 
 
+## Other Language Support
+
+[Turkish](README_TU.md)
```
Comment (PaParaZz1) [inline README.md:317]: also add `CONTRIBUTING_TU.md`

File: README_TU.md [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+
+## Katkı
+
+Amacımız bu depoyu daha da iyi hale getirmek. Katkıda bulunmakla ilgileniyorsanız, katkıyla ilgili talimatlar için lütfen [BURAYA](CONTRIBUTING_TU.md) bakın.
+
+## Lisans
+
+Müthiş RLHF, Apache 2.0 lisansı altında yayınlandı.
+[01Kevin01](https://github.com/01Kevin01)
+
```
Comment (PaParaZz1) [inline README_TU.md]: remove empty lines

**Review by PaParaZz1 — APPROVED** [review]

File: README_TU.md (hunk 2 above)
Comment (ruoyuGao) [reply README_TU.md]: removed

File: README.md (hunk 1 above)
Comment (ruoyuGao) [reply README.md:317]: it was already used in README_TU.md

**Pull request merged**